README.md

```diff
@@ -44,11 +44,17 @@ You need to install autoconf, automake and libtool:
 
 	brew install autoconf automake libtool
 	
+In case OpenSSL is not installed:
+    
+	brew install openssl
+	cd /usr/local/include
+	ln -s ../opt/openssl/include/openssl .
```

Comment:
We should not require the user to link brew's OpenSSL into /usr/local, as this might override other preferences they have.  (Also, the command above only links in the includes, but not the libraries, so it could lead to inconsistencies.)  

Instead, we want to get the following environment variables added.  (See the instructions in `brew info openssl`.)

```
LDFLAGS:  -L/usr/local/opt/openssl/lib
CPPFLAGS: -I/usr/local/opt/openssl/include
```

So we need to find some way of telling configure to look there when on macOS, but I don't know autoconf well enough to know how to tell it to do that.  @smashra? @tlepoint?